Firmware/ultralcd.cpp

```diff
@@ -2209,34 +2209,35 @@ void mFilamentItem(uint16_t nTemp, uint16_t nTempBed)
 
             lcd_clear();
             lcd_draw_update = 1;
-            lcd_puts_at_P(0, 3, _i(">Cancel")); ////MSG_ c=20 r=1
+            lcd_puts_at_P(0, 3, _T(MSG_CANCEL)); ////MSG_CANCEL
 
             lcd_set_cursor(0, 1);
             switch (eFilamentAction)
             {
             case FilamentAction::Load:
             case FilamentAction::AutoLoad:
             case FilamentAction::MmuLoad:
-                lcd_puts_P(_i("Preheating to load")); ////MSG_ c=20
+                lcd_puts_P(_i("Preheating to load")); ////MSG_PREHEATING_TO_LOAD c=20
                 raise_z_above(MIN_Z_FOR_LOAD);
                 break;
             case FilamentAction::UnLoad:
             case FilamentAction::MmuUnLoad:
-                lcd_puts_P(_i("Preheating to unload")); ////MSG_ c=20
+                lcd_puts_P(_i("Preheating to unload")); ////MSG_PREHEATING_TO_UNLOAD c=20
                 raise_z_above(MIN_Z_FOR_UNLOAD);
                 break;
             case FilamentAction::MmuEject:
-                lcd_puts_P(_i("Preheating to eject")); ////MSG_ c=20
+                lcd_puts_P(_i("Preheating to eject")); ////MSG_PREHEATING_TO_EJECT c=20
                 break;
             case FilamentAction::MmuCut:
-                lcd_puts_P(_i("Preheating to cut")); ////MSG_ c=20
+                lcd_puts_P(_i("Preheating to cut")); ////MSG_PREHEATING_TO_CUT c=20
                 break;
             case FilamentAction::None:
             case FilamentAction::Preheat:
             case FilamentAction::Lay1Cal:
                 // handled earlier
                 break;
             }
+            lcd_puts_at_P(0, 3, _T(MSG_CANCEL)); ////MSG_CANCEL
```

Comment:
why was this line added?

Reply:
copy/paste error fixed in https://github.com/prusa3d/Prusa-Firmware/pull/3079/commits/30b383195dbcd2bfea94c074a746e0b99005ab38